segmented_sieve_of_eratosthenes/segmentedSieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+#include <vector>
+using namespace std;
+
+vector<int> segmentedSieve(int segMax) {
```

Comment:
Please document the function, see https://github.com/iiitv/algos/blob/master/binary_search/binary_search.c for reference.